src/containers/userInfo.js

```diff
@@ -0,0 +1,39 @@
+import React, { Component } from 'react'
+import superagent from 'superagent/lib/client'
+
+const api_key = ''
+const token = "eyJhbGciOiJIUzUxMiIsInR5cCI6IkpXVCJ9.eyJhdWQiOiJzYWZpcmEiLCJleHAiOjE1NDQ3MTE1MDEsImlhdCI6MTU0MjI5MjMwMSwiaXNzIjoic2FmaXJhIiwianRpIjoiYjE1Y2Y3NTAtMzQzNi00NDUxLWExMjAtZGM1Y2NkOWNkYzZiIiwibmJmIjoxNTQyMjkyMzAwLCJzdWIiOiIxIiwidHlwIjoiYWNjZXNzIn0.dVlx-TL5f0_g5s-oUUIcX-XDMl3ofatzNbWPJdlBrI-q_y00RNpcUlnZh6xxMUB_S6zfse7uSPjSIlAYi_9tJw"
+const api_endpoint = 'https://sei19-safira.herokuapp.com/api/v1/user'
```

Comment:
Use the .env file.

Reply:
Not sure how to do this without installing dotenv package. @Mendess2526, how did you handle this file?